src/Resource.js

```diff
@@ -12,7 +12,7 @@ const getOptionsQuery = (optionsObject = {}) =>
 const relatedResourceUrl = ({ parent, relationship }) =>
   `${parent.type}/${parent.id}/${relationship}`;
 
-const extractData = response => response.data;
+const extractData = response => Object.assign(response.data, response.included ? { included: response.included } : null);
```

Comment:
The idea of having access to the `included` data certainly makes sense. I don't think this change is necessary, though. I did some digging and confirmed that `@reststate/client` returns the entire response from the server, including `data` and `included` properties and anything else. I've added some detail to a test confirming this:

https://github.com/reststate/reststate-client/pull/15/commits/d778e8d92ca448fcbcc90c4e40ebb3d2af1b1870

The test shows that if the API returns:

```
{
  data: {
    data: {…},
    included: {…},
  }
}
```

Then `@reststate/client` will return:

```
{
  data: {…},
  included: {…},
}
```

Note that the `data` property that `@reststate/client` removes is the `data` property of the object returned by Axios, which represents all of the response data. It just so happens that JSON:API also has a `data` property in its returned data, at the same level as `included`; that certainly leads to some confusion in my own code. If you're using an adapter other than Axios, you will want to set it up to return objects with a `data` property wrapping the whole response, to be compatible with Axios.

Reply:
More info: I checked out your fork of `@reststate/mobx` and I think what's going on is that, for example on line 49, the `data` property of the `response` is accessed, so that's where the `included` property is being skipped over. You might need to make more extensive changes through `@reststate/mobx` to account for the `included` property in some way—especially because it's the included data for _all_ response records in case multiple are returned.

I'm excited someone is looking into handling included data—it's been a challenging thing that I haven't been able to figure out an approach for! I look forward to keeping in touch about it.

Reply:
@reststate/vuex should be changed for this intention. (in your case @reststate/mobx should)

Every action extracts data from response "data". Currently, the client returns a response formed as:

```
{
  data: { ... }, // extracted from store
  included: [ ... ], // not extracted currently
}
```

So, every store action regarding storing records should use the response as it is. Then all records can contain included data.